pkg/kubectl/cmd/describe.go

```diff
@@ -76,29 +83,27 @@ func RunDescribe(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []s
 	if err != nil {
 		return err
 	}
-	mapping, err := r.ResourceMapping()
-	if err != nil {
-		return err
-	}
 
-	describer, err := f.Describer(mapping)
-	if err != nil {
-		return err
-	}
-	infos, err := r.Infos()
-	if err != nil {
-		if errors.IsNotFound(err) && len(args) == 2 {
-			return DescribeMatchingResources(mapper, typer, describer, f, cmdNamespace, args[0], args[1], out)
```

Comment:
`DescribeMatchingResources` tries to find resources that match name prefix. We should keep it somewhere.